website/docs/guides/test-your-typegraph/index.mdx

```diff
@@ -0,0 +1,430 @@
+import TabItem from "@theme/TabItem";
+import SDKTabs from "@site/src/components/SDKTabs";
+
+import PyPmTabs from "@site/src/components/PythonPackageManagerTabs";
+import TsPmTabs from "@site/src/components/TsPackageManagerTabs";
+
+# Test typegraphs
+
+[Typegraph](/docs/reference/typegraph) is the SDK for _Metatype_,  used to author and define applications..
```

Comment:
nitpick: I'd rework this sentence as the typegraph is primarily the data/programming model wheras the SDK is much more including metagen, the test stuff..etc.